src/app/blog/[slug]/helpers.tsx

```diff
@@ -0,0 +1,38 @@
+import { lstatSync, readdirSync } from "fs";
+import path from "path";
+
+export interface MarkdownFileProps {
+  default: {
+    data: { [key: string]: string };
+    content: string;
+  };
+}
+
+export interface GetMarkdownFileProps {
+  frontmatter: { [key: string]: string };
+  markdownBody: string;
+}
+
+export const blogPostsPath = path.resolve(process.cwd(), "./src/posts/");
+
+export function isFile(fileName: string) {
+  return lstatSync(fileName).isFile();
+}
+
+export function formatPathAsUrl(filePath: string, fileType: string) {
+  return path.basename(filePath, fileType).replace(/ /g, "-");
+}
+
+export function getBlogPosts(folderPath: string) {
```

Comment:
🔧 Small naming nit. This gets the file name of blog posts. It accepts a path to a folder and returns a string of arrays.`getBlogPosts` makes it sound like this function returns a file.

Reply:
Something like:

* `getFileNames`

Technically, this util could be used for any file path that's a folder.